### PR TITLE
Settings page added

### LINKED ISSUE #7: Settings page
### Description

<b>create seperate branch named-> <yourname>-page or component name you are working on</b>

Component should be able to take all the values from params / constructor for all dynamic fields.
Proper validation should be done for each field.

### Preview

image

### PR DESCRIPTION
**Pull request by Sayakd915**
commit -m "Settings Page Added"

https://github.com/user-attachments/assets/33c5fce9-86e0-45bc-a734-447aa91fd8f5

fix #26 
fix #7

### CODE REVIEW

File: lib/app/modules/Settings/views/settings_view.dart [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                colors: [theme.colorLevel1, theme.colorLevel2],
+                begin: Alignment.topCenter,
+                end: Alignment.bottomCenter,
+              ),
+            ),
+          ),
+          SingleChildScrollView(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
```
Comment (SHREYANSHSK) [inline lib/app/modules/Settings/views/settings_view.dart]: give mainAxis alignment and make the page responsive such that no need to scroll this page

**Review by SHREYANSHSK — CHANGES_REQUESTED** [review]
make this small change after that i will merge the pr. and also resolve the conflicts

**Comment by Sayakd915** [comment]
commit -m "Settings Page Edited"

fix #7 


https://github.com/user-attachments/assets/1fd88263-f3a6-435f-9c3d-4cf175c6f539

**Review by SHREYANSHSK — APPROVED** [review]
merging the pr

**Pull request merged**